front/lib/api/workspace.ts

```diff
@@ -415,63 +398,23 @@ export async function changeWorkspaceName(
   owner: LightWorkspaceType,
   newName: string
 ): Promise<Result<void, Error>> {
-  const [affectedCount] = await WorkspaceModel.update(
-    { name: newName },
-    {
-      where: {
-        id: owner.id,
-      },
-    }
-  );
-
-  if (affectedCount === 0) {
-    return new Err(new Error("Workspace not found."));
-  }
-
-  return new Ok(undefined);
+  return WorkspaceResource.updateName(owner.id, newName);
 }
 
 export async function updateWorkspaceConversationsRetention(
   owner: LightWorkspaceType,
   nbDays: number
 ): Promise<Result<void, Error>> {
-  const [affectedCount] = await WorkspaceModel.update(
-    { conversationsRetentionDays: nbDays === -1 ? null : nbDays },
-    {
-      where: {
-        id: owner.id,
-      },
-    }
-  );
-
-  if (affectedCount === 0) {
-    return new Err(new Error("Workspace not found."));
-  }
-
-  return new Ok(undefined);
+  return WorkspaceResource.updateConversationsRetention(owner.id, nbDays);
 }
 
 export async function disableSSOEnforcement(
```

Comment:
Does it make sense to keep this method at all, vs just call `WorkspaceResource.disableSSOEnforcement(workspace.id)` directly from whoever calls this? It's not adding a lot of value...

Reply:
Ok, I yanked a few proxies.